app/src/components/recent-searches.tsx

```diff
@@ -0,0 +1,122 @@
+import { Box, Text } from "@chakra-ui/react";
+import React from "react";
+
+const RecentSearches = () => {
+  const hasRecentSearches = true;
+  return (
+    <Box>
+      <Box className="px-4">
+        <Text
+          color="content.tertiary"
+          fontWeight="extrabold"
+          lineHeight="16px"
+          letterSpacing="widest"
+          fontSize="overline"
+          fontFamily="heading"
+        >
+          RECENT SEARCHES
+        </Text>
+      </Box>
+      {hasRecentSearches ? (
+        <Box>
+          <ul className="list-none">
+            <li>
+              <Box className="h-[72px] w-full flex flex-col justify-center group px-4 hover:bg-[#2351DC] transition-all duration-150 cursor-pointer">
+                <Text
+                  className="group-hover:text-white"
+                  color="content.secondary"
+                  fontSize="body.lg"
+                  fontFamily="body"
+                  fontWeight="normal"
+                  lineHeight="24"
+                  letterSpacing="wide"
+                >
+                  Rio de Janeiro
```

Comment:
Why is this hardcoded? If you don't have the data for this yet, you can have an array at the top of the component and render the entries here. Otherwise let's save the recent entries into local storage.
One way to do that would be to use redux-persist:
https://blog.logrocket.com/persist-state-redux-persist-redux-toolkit-react/